Testing/ORKTest/ORKTest/Tasks/TaskFactory+QuestionSteps.m

```diff
@@ -677,6 +683,35 @@ @implementation TaskFactory (QuestionSteps)
     }
     
     {
+        /*
+         Horizontal text choice scale with visible selected value.
+         */
+        ORKTextChoice *textChoice1 = [ORKTextChoice choiceWithText:@"Poor" value:@(1)];
+        ORKTextChoice *textChoice2 = [ORKTextChoice choiceWithText:@"Fair" value:@(2)];
+        ORKTextChoice *textChoice3 = [ORKTextChoice choiceWithText:@"Good" value:@(3)];
+        ORKTextChoice *textChoice4 = [ORKTextChoice choiceWithText:@"Above Average" value:@(4)];
+        ORKTextChoice *textChoice5 = [ORKTextChoice choiceWithText:@"Excellent" value:@(5)];
+        
+        NSArray *textChoices = @[textChoice1, textChoice2, textChoice3, textChoice4, textChoice5];
+        
+        ORKTextScaleAnswerFormat *scaleAnswerFormat = [ORKAnswerFormat textScaleAnswerFormatWithTextChoices:textChoices
+                                                                                               defaultIndex:3
+                                                                                                   vertical:NO];
+        
+        scaleAnswerFormat.hideSelectedValue = YES;
+        
```

Comment:
This is not consistent with the comment at the beginning of the block -         
          /*
          Horizontal text choice scale with visible selected value.
          */

Update the comment?

Reply:
Sorry about that.